draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -43,17 +44,25 @@ normative:
 informative:
   I-D.steele-cose-merkle-tree-proofs: COMTRE
 
+  CCF:
+    title: "Confidential Consortium Framework"
+    target: "https://github.com/microsoft/ccf"
+
 --- abstract
 
-This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for implementations that rely on Trusted Execution Environments (TEEs) to provide stronger tamper-evidence guarantees.
+This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for transaction ledgers produced by Trusted Execution Environments (TEEs), such as the Confidential Consortium Framework ({{CCF}}) to provide stronger tamper-evidence guarantees.
 
 --- middle
 
 # Introduction
 
-The Concise Encoding of Signed Merkle Tree Proofs (CoMeTre) {{-COMTRE}} defines a common framework for defining different types of proofs, such as proof of inclusion, about verifiable data structures (also abbreviated as "logs" in this document). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the log, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the log at a later time.
+The Concise Encoding of Signed Merkle Tree Proofs (CoMeTre) {{-COMTRE}} defines a common framework for defining different types of proofs, such as proof of inclusion, about verifiable data structures (VDS). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the VDS, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the VDS at a later time.
```

Comment:
Maybe:
```suggestion
The COSE Receipts document {{-COMTRE}} defines a common framework for defining different types of proofs, such as proof of inclusion, about verifiable data structures (VDS). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the VDS, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the VDS at a later time.
```
Also, shouldn't I-D.steele-cose-merkle-tree-proofs (`{{-COMETRE}}`)be a normative reference?

Reply:
Err... I-D.**IETF**-cose-merkle-tree-proofs I mean, of course :sweat_smile:

Reply:
Will fix